apps/bsd/src/screens/BallotEjectScreen.tsx

```diff
@@ -247,6 +305,8 @@ const BallotEjectScreen = ({
     !isInvalidPrecinctSheet &&
     !isUnreadableSheet
 
+  const isBlankSheet = frontIsBlank && backIsBlank
```

Comment:
Is this used? I can't tell from here on my iPhone review screen.

Reply:
Yes it's used at the bottom right before return.